Cabal/Cabal.cabal

```diff
@@ -410,6 +480,9 @@ library
     Distribution.Types.ComponentRequestedSpec
     Distribution.Types.TargetInfo
     Distribution.Utils.NubList
+    Distribution.Utils.UnionFind
+    Distribution.Utils.Base62
+    Distribution.Utils.Progress
```

Comment:
Would this be a case for an explicit "Internal" module space? Seems it would be a good idea for cases where things have to be exposed for "incidental" reasons.